docs/get-started.md

```diff
@@ -6,7 +6,9 @@ For simplicity, the example uses Minio, an S3-compatible storage service that ru
 
 **NOTE** The example lets you explore basic Ark functionality. In the real world, however, you would back your cluster up to external storage.
 
-See [Set up Ark on your platform][3] for how to configure Ark for a production environment. 
+See [Set up Ark on your platform][3] for how to configure Ark for a production environment.
+
+If you run Minikube for your cluster, you must also perform some additional steps. See [Minikube configuration][2]. Note that you'll need to make similar modifications to view the Minio logs.
```

Comment:
So, couple things here - (1) the additional steps are not Minikube-specific; this will be necessary for anyone using this "get started" guide.  The additional steps are needed *because minio is running inside the cluster.* . (2) since they'll always be needed, we should revise our example YAML here (make the minio service `NodePort` by default; update `05-ark-backupstoragelocation.yaml` to have an `s3Url` placeholder that implies it needs to be reachable from wherever the `ark` client is running).  Really the only minikube-specific thing here is **how** to get that externally-reachable `s3Url`.